--- a/drivers/net/Kconfig
+++ b/drivers/net/Kconfig
@@ -87,6 +87,8 @@ config WIREGUARD
 	  If you're build this for a desktop, say Y.
 	  If you're build this for a phone, say Y.
 
+source "drivers/net/wireguard/Kconfig"
+
 --- a/drivers/net/Makefile
 +++ b/drivers/net/Makefile
@@ -12,6 +12,7 @@ obj-$(CONFIG_MDIO_DEVICE) += mdio.o
 obj-$(CONFIG_NET_UDP_TUNNEL) += udp_tunnel.o
+obj-$(CONFIG_WIREGUARD) += wireguard/
